src/NHSD.BuyingCatalogue.Solutions.API/Controllers/AdditionalServiceController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Linq;
+using System.Net.Mime;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Solutions.API.ViewModels.AdditionalService;
+
+namespace NHSD.BuyingCatalogue.Solutions.API.Controllers
+{
+    [Route("api/v1/additional-services")]
+    [ApiController]
+    [Produces(MediaTypeNames.Application.Json)]
+    public sealed class AdditionalServiceController : ControllerBase
+    {
+        [HttpGet]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "Cannot be viewed on swagger if static")]
+        public ActionResult<AdditionalServiceResult> Get([FromQuery] string[] solutionIds)
+        {
+            return new AdditionalServiceResult
+            {
+                Id = solutionIds.FirstOrDefault(),
```

Comment:
Can you make this a valid looking Additional Service ID please? definition from Confluence:
```
Concatenation of the Solution Id following by a number incremented for each new additional service of that solution, in the form
[Solution Id] + 'A' + [Id]
Where
[Id] = an zero padded sequence number, 3 characters long.
Example:
10000-001A001
10000-001A002
```

Reply:
Just pushed the changes for this 👍